source/expressions.tex

```diff
@@ -1927,7 +1927,7 @@
 operators otherwise\iref{class.copy.assign}.
 \begin{note}
 These special member functions are implicitly defined as
-usual, and might therefore be defined as deleted.
+usual, and can therefore be defined as deleted.
```

Comment:
This sounds uncomfortably ambiguous with the meaning of "and therefore you the user can define it as deleted". The previous wording wasn't quite so readily interpretable that way. Maybe extra words like "and can therefor end up as being defined as deleted". (Maybe for 23.)

Reply:
How about:
```suggestion
usual, which can result in them being defined as deleted.
```

Reply:
Great, another example to go on the "noteworthy examples" list!